From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Fri, 1 May 2015 17:48:10 -0400
Subject: shareLib: fix for mingw

For some reason the _DLL macro sometimes gets defined
event for static mingw builds.  The logic of
sharedLib.h causes dllimports w/o matching dllexports
and linking failure.
---
 src/libCom/misc/shareLib.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libCom/misc/shareLib.h b/src/libCom/misc/shareLib.h
index ba23b28..ca251d8 100644
--- a/src/libCom/misc/shareLib.h
+++ b/src/libCom/misc/shareLib.h
@@ -126,7 +126,7 @@
 #           define epicsShareFunc  __declspec(dllexport)
 #       endif
 #   else
-#       if defined(_DLL) /* this indicates that we are being compiled to call DLLs */
+#       if defined(_DLL) && !defined(EPICS_DLL_NO) /* this indicates that we are being compiled to call DLLs */
 #           define epicsShareExtern __declspec(dllimport) extern 
 #           define epicsShareClass  __declspec(dllimport) 
 #           define epicsShareFunc  __declspec(dllimport)
